STAR/draft-ietf-acme-star.md

```diff
@@ -347,7 +355,8 @@ When omitted, the start date is as soon as authorization is complete.
 - recurrent-end-date (required, string): the latest date of validity of the last certificate issued,
 in {{RFC3339}} format.
 - recurrent-certificate-validity (required, integer): the maximum validity period of each STAR certificate, an integer that denotes a number of seconds.  This is a nominal value which does not include any
-extra validity time which is due to pre-dating.  The client can use this value as a hint to configure its polling timer.
+extra validity time which is due to pre-dating.  The client can use the value reflected by the server as a hint to configure its polling timer.
```

Comment:
clarify: reflected by the server (which may be different from the one sent by the client)

Reply:
Sure.  (Fixed in e42ec00)